src/test/java/testQuotes/QuoteControllerTest.java

```diff
@@ -30,6 +32,7 @@
 import org.springframework.web.util.UriComponents;
 import org.springframework.web.util.UriComponentsBuilder;
 
+@ActiveProfiles("dev")
```

Comment:
Remove the annotation. The unit tests should be executed in every environment

Reply:
Exactly this annotation makes it so. @ActiveProfiles("dev") annotation is for tests only and executes the build with the "dev" profile environment variables no matter the current active profile. If that annotation is not here with "prod" profile active, the tests would be ran during maven compile without active mysql server, which is started via docker-compose after the jar is already complete.

Reply:
ok, the integration tests will be configured as future task